components/latex_block/latex_block.tsx

```diff
@@ -37,19 +34,22 @@ export default class LatexBlock extends React.PureComponent {
         }
 
         try {
-            const katexOptions = {
+            const katexOptions: KatexOptions = {
                 throwOnError: false,
                 displayMode: true,
                 maxSize: 200,
                 maxExpand: 100,
                 fleqn: true,
             };
-            const html = this.state.katex.renderToString(this.props.content, katexOptions);
+
+            this.setState({
```

Comment:
We shouldn't be calling `setState` in a render function, this will cause an infinite loop.
What I meant originally was to assign the function itself to state, rather than the output of `renderToString`